swagger_to_sdk_config.json

```diff
@@ -1,10 +1,9 @@
 {
   "$schema": "https://openapistorageprod.blob.core.windows.net/sdkautomation/prod/schemas/swagger_to_sdk_config.schema.json",
   "meta": {
-    "after_scripts": [
+    "after_scripts_in_repo": [
```

Comment:
How does `after_scripts_in_repo` change things?

Reply:
This is per what Phoenix recommends, to emphasize this script runs in the root dir and only run once, even there are multiple changed services